rerun_py/src/server.rs

```diff
@@ -0,0 +1,108 @@
+use std::net::SocketAddr;
+
+use pyo3::{
+    Bound, PyResult, Python,
+    exceptions::PyValueError,
+    pyclass, pymethods,
+    types::{
+        PyAnyMethods as _, PyDict, PyDictMethods as _, PyModule, PyModuleMethods as _, PyString,
+        PyStringMethods as _,
+    },
+};
+use re_sdk::external::re_server::{self, Args as ServerArgs};
```

Comment:
Any reason you included `re_server` in `re_sdk/Cargo.toml` and not in `rerun_py/Cargo.toml`?